### ISSUE TITLE
PHP version requirement

### ISSUE THREAD

**Issue by smaiLee** [Martin-Gleiss/smartvisu#142]
At the moment develop version requires php 5.6.1.
Either document this or (better) ensure compatibility to 5.4 as before.

**Comment by Tom-Bom-badil** [comment]
Just a remark: See [here](https://github.com/Martin-Gleiss/smartvisu/issues/141) - there seem to be compatibility issues with php 7, too. So for the time being, develop only works with php 5.6.

**Comment by smaiLee** [comment]
You're right, php 7 has to be supported.
@lolouk44 which issues did you have on php 7.0?

**Comment by lolouk44** [comment]
To be honest I would not use my issues as a generic reason to stick to PHP5.x.
I upgraded from Ubuntu 14.04 to 16.04 and experienced many issues, but I would bet it's down to something I've done rather than the upgrade or PHP7.0 itself,
My issue was that after the upgrade I could not access any of my webpages (SmartVISU, but also Wordpress etc.)
I did notice that on some of my search results (just can't retrieve the actual site) someone said there was a known issue with PHP7.0 and therefore he needed to downgrade to PHP5.6.
Interestingly after I downgraded, everything started to work.
I will eventually try and have another go at php7.0 but got a few issues to sort first.
So to sum up, "ignore me and my issues" if you consider (and you should) supporting PHP7.0

**Comment by smaiLee** [comment]
I just switched to php 7.0.15 and did a very brief test just against offline driver and services.
At least this did work without any issues.

If someone likes to test deeper, he would be very welcome.
Otherwise I would not take any further action, as I don't see any reason why php 7.0 should not work.

**Comment by ThomasCr** [comment]
I am also on Ubuntu 16.04 with PHP 7.
It has needed some time to figure out how to get php-fpm on apache running. But I had no problems running smartVISU (2.8 and 2.9) on it.

**Comment by lolouk44** [comment]
Thanks, I'll have another go at it in the next few days. Who knows it might even resolve my issue with the plots (crossing fingers)

**Comment by smaiLee** [comment]
Add mbstring Module to requirements (was activated by default in PHP 5 but not anymore in PHP 7)
